css/home.css

```diff
@@ -1,4 +1,52 @@
 /* Home page specific styles */
 
 
```

Comment:
for the whole file please use soft tab (2 spaces) to indent as suggested by our style guide
https://github.com/airbnb/css#formatting